README.md

```diff
@@ -596,6 +596,27 @@ test('renders text', function(assert) {
 
 **Note: you could create a helper and then use it like `Resolver from ../helpers/appointments-manager/resolver`**
 
+### Testing for standalone engines
+
+If you have an async engine, you'll need to edit your `tests/test-helper.js` like this:
+
+```js
+import resolver from './helpers/resolver';
+import {
+  setResolver
+} from 'ember-qunit';
+import preloadAssets from 'ember-asset-loader/test-support/preload-assets';
+import manifest from 'dummy/config/asset-manifest';
+import { start } from 'ember-cli-qunit';
+
+preloadAssets(manifest); // This ensures all engine resources are loaded before the tests
+setResolver(resolver);
+start();
+```
+
+This should be enough to make integration & acceptance tests work.
+For unit tests, you'll need to use a custom resolver, as described in [Unit/Integration testing for standalone engines](#unitintegration-testing-for-in-repo-engines).
```

Comment:
Since this is a standalone Engine, you should be able to replace the global resolver (set with `setResolver`) to just use the Engine resolver instead, and then you avoid needing a custom resolver for all your unit tests.

Reply:
I'm not sure I understand this.

In my standalone, lazy loading engine, I had the following setup:

```js
// tests/test-helper.js
import resolver from './helpers/resolver';
import {
  setResolver
} from 'ember-qunit';
import preloadAssets from 'ember-asset-loader/test-support/preload-assets';
import manifest from 'dummy/config/asset-manifest';
import { start } from 'ember-cli-qunit';

setResolver(resolver);
preloadAssets(manifest).then(start);
```

```js
// tests/helpers/resolver.js
import Resolver from '../../resolver';
import config from '../../config/environment';

const resolver = Resolver.create();

resolver.namespace = {
  modulePrefix: config.modulePrefix,
  podModulePrefix: config.podModulePrefix
};

export default resolver;
```

And that didn't work for me, in my unit tests. (All other tests, e.g. integration & acceptance, work). What would I need to change here?